.gitignore

```
@@ -1 +1,3 @@
 **/target
+node_modules
+package-lock.json
```

You might prefer something like this to avoid these files being populated anywhere:
```suggestion
**/node_modules
**/package-lock.json
```